platform/packages/currency/src/group.rs

```diff
@@ -1,8 +1,12 @@
+use std::fmt::Debug;
+
+use sdk::schemars::JsonSchema;
+
 use crate::SymbolSlice;
 
 use super::{matcher::Matcher, AnyVisitor, AnyVisitorResult};
 
-pub trait Group: PartialEq {
+pub trait Group: Debug + Copy + Eq + JsonSchema + 'static {
```

Comment:
two things:
1. why do we impose more constraints adding more super-traits?
2. not agree on depending to `sdk` at this core module

Reply:
I wouldn't have if all the implementations didn't require them. If you are fine with converting them to manual implementations instead of `#[derive(...)]` ones, there won't be a need for that.

The whole problem is that the derived implementations place those bounds on all generic types, not on the fields themselves.

Reply:
About (2), I'd like to ask why was it added in the first place, if we don't want it? I can remove it, but perhaps we'd like to explore ways to avoid `schemars`'s problems which I encountered during the refactoring. Another possibility is to drop it entirely and maybe find a better suiting one.